examples/veins/antenna.xml

```diff
@@ -1,3 +1,23 @@
+<!--
+Copyright (C) 2016 Alexander Brummer <alexander.brummer@fau.de>
+
+Documentation for these modules is at http://veins.car2x.org/
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
```

Comment:
how would you feel about adding a third option, allowing modification and redistribution under the terms of CC-by-sa?

Reply:
Sure, I'm fine with it.